tests/probe/t14-specimen.sh

```
@@ -0,0 +1,23 @@
+# Throwaway specimen for the T14 probe (issue #59). Deliberately defective so
+# CodeRabbit has something to flag and something to autofix. Delete with the
+# branch.
+
+collect_logs() {
+  DIR=$1
+  cd $DIR
+  for f in `ls`; do
+    if [ $f == "skip" ]; then
+      continue
+    fi
+    cat $f | grep "ERROR" > /tmp/errors.txt
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)